File: momentra/momentra/Shell/BusinessEmpty/BusinessSetupKind.swift
import SwiftUI

enum BusinessSetupKind: String, CaseIterable, Identifiable {
    case teamOperations = "TEAM_OPERATIONS"
    case businessRunway = "BUSINESS_RUNWAY"
    case businessOperations = "BUSINESS_OPERATIONS"

    var id: String { rawValue }

    var title: String {
        switch self {
        case .teamOperations: return "Set up Team Operations"
        case .businessRunway: return "Set up Business Runway"
        case .businessOperations: return "Set up Business Operations"
        }
    }

    var familyCode: String { rawValue }

    var analyticsScreen: String {
        switch self {
        case .teamOperations: return AnalyticsScreens.businessSetupTeamOps
        case .businessRunway: return AnalyticsScreens.businessSetupRunway
        case .businessOperations: return AnalyticsScreens.businessSetupOps
        }
    }

    var activateColor: Color {
        switch self {
        case .teamOperations: return Color(hex: "#10B981")
        case .businessRunway: return Color(hex: "#FBBF24")
        case .businessOperations: return Color(hex: "#818CF8")
        }
    }

    var maestroTag: String {
        switch self {
        case .teamOperations: return "business.setup.team_operations"
        case .businessRunway: return "business.setup.business_runway"
        case .businessOperations: return "business.setup.business_operations"
        }
    }

    var ctaGradient: LinearGradient {
        switch self {
        case .teamOperations:
            return LinearGradient(colors: [Color(hex: "#10B981"), Color(hex: "#34D399")], startPoint: .leading, endPoint: .trailing)
        case .businessRunway:
            return LinearGradient(colors: [Color(hex: "#FBBF24"), Color(hex: "#F59E0B")], startPoint: .leading, endPoint: .trailing)
        case .businessOperations:
            return LinearGradient(colors: [SetupTokens.bizAccent, SetupTokens.accentPurple], startPoint: .leading, endPoint: .trailing)
        }
    }
}

File: apk/app/src/main/java/com/example/momentra/ui/shell/empty/BusinessSetupKind.kt
package com.example.momentra.ui.shell.empty

import androidx.compose.ui.graphics.Color
import com.example.momentra.analytics.AnalyticsScreens
import com.example.momentra.ui.shell.maestro.MaestroIds

enum class BusinessSetupKind(
    val title: String,
    val familyCode: String,
    val analyticsScreen: String,
    val activateColor: Color,
    val maestroTag: String,
) {
    TEAM_OPERATIONS(
        "Set up Team Operations",
        "TEAM_OPERATIONS",
        AnalyticsScreens.BUSINESS_SETUP_TEAM_OPS,
        Color(0xFF10B981),
        MaestroIds.BUSINESS_SETUP_TEAM,
    ),
    BUSINESS_RUNWAY(
        "Set up Business Runway",
        "BUSINESS_RUNWAY",
        AnalyticsScreens.BUSINESS_SETUP_RUNWAY,
        Color(0xFFFBBF24),
        MaestroIds.BUSINESS_SETUP_RUNWAY,
    ),
    BUSINESS_OPERATIONS(
        "Set up Business Operations",
        "BUSINESS_OPERATIONS",
        AnalyticsScreens.BUSINESS_SETUP_OPS,
        Color(0xFF818CF8),
        MaestroIds.BUSINESS_SETUP_OPS,
    ),
}
